Makefile

```diff
@@ -76,19 +76,43 @@ ctl:
 run:
 	cargo run --no-default-features --features  "${ENABLE_FEATURES}" --bin tikv-server
 
+# An optimized build suitable for development and benchmarking
 release:
+	make dist_release
+
+# An optimized build with additional SSE4.2 optimizations
```

Comment:
I think all release builds have the sse4.2 optimisations. The unportable bit is specific to RocksDB (maybe SSE4.2 there? I'm not sure)

Reply:
Yes you are right. The release target turns on "jemalloc portable sse no-fail". unportable_turns of the "portable" feature, which I don't know the purpose of. I'll have to look into this and update the comments.

Reply:
Interesting that there is no target that excludes the "sse" feature.